README.md

```diff
@@ -13,12 +13,32 @@ Create and manage Azure Virtual Machines directly from VS Code.
 
 ### Create New Virtual Machine
 
-![CreateVirtualMachine](resources/CreateVirtualMachine.gif)
+- Create an Azure VM with the default image of "Ubuntu 18.04-LTS" and size "Standard D2s V3"
+
+![CreateVirtualMachine](resources/CreateAzureVM.png)

```

Comment:
The casing of `png` seems to be off causing some problems here. Also I'd prefer if you put all of these into a `resources/readme/` folder that is excluded in `.vscodeignore` so that we're not copying these to everyone's machine